### PR TITLE
DOC: update pytest line

### PR DESCRIPTION
**Pull request by raybellwaves**
Closes https://github.com/zarr-developers/zarr-python/issues/922

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -97,7 +97,7 @@ the repository, you can do something like the following::
 
 To verify that your development environment is working, you can run the unit tests::
 
-    $ pytest -v zarr
+    $ python -m pytest -v zarr
```
Comment (jakirkham) [inline docs/contributing.rst:100]: There is another line like this further down. Should we update that as well?

Any other command line calls like this we should update?

**Review by jakirkham — COMMENTED** [review]
Thanks Ray! 😄

Couple questions below

File: docs/contributing.rst (hunk 1 above)
Comment (joshmoore) [reply docs/contributing.rst:100]: @raybellwaves ?

**Comment by joshmoore** [comment]
"[more python -m](https://github.com/zarr-developers/zarr-python/pull/923/commits/babe56deeadf6b1a84e4b61c5d8a8d1a2dec6dcf)" indeed! Thanks, @raybellwaves! I'm going to go ahead and get this in for the imminent 2.11.

**Pull request merged**